test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/JsonData.cs

```diff
@@ -4088,5 +4088,13 @@ public static class JsonData
           }";
         #endregion
 
+        #region TempApiKeyResponse
+        public const string tempApiKeyJsonData = @"{
+            ""Key"": ""tempkey"",
+            ""Expires"": ""/Date(1489106836905)/"",
```

Comment:
I think Christy make the change for this -- can we modify the date to be:
`"2017-03-09T16:47:16"`